Bottom: bdbe022df87a396977b63e070bbb159443056eb4
Top:    1ee2df6e314eb7110470843734d729ab71a9db8c
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:58 -0500

Fix buildsystem credentials entry


---
diff --git a/ansible-playbook/group_vars/all/credentials.yml b/ansible-playbook/group_vars/all/credentials.yml
index 04b0a1ac..7c9bb550 100644
--- a/ansible-playbook/group_vars/all/credentials.yml
+++ b/ansible-playbook/group_vars/all/credentials.yml
@@ -15,6 +15,6 @@ mantisbt_email_password:             '{{ lookup("password", inventory_dir + "/cr
 buildsystem_ssh_prod_user_password:  '{{ lookup("password", inventory_dir + "/credentials/buildsystem.novatech-llc.com/ssh/prod length=20") }}'
 buildsystem_ssh_root_user_password:  '{{ lookup("password", inventory_dir + "/credentials/buildsystem.novatech-llc.com/ssh/root length=20") }}'
 buildsystem_database_root_password:  '{{ lookup("password", inventory_dir + "/credentials/buildsystem.novatech-llc.com/database/root length=20") }}'
-buildsystem_database_build_password: '{{ lookup("file",     inventory_dir + "/credentials/buildsystem.novatech-llc.com/database/readuser") }}'
+buildsystem_database_build_password: '{{ lookup("password", inventory_dir + "/credentials/buildsystem.novatech-llc.com/database/readuser") }}'
 svn_username:                        '{{ lookup("file",     inventory_dir + "/personal_credentials/svn/username") }}'
 svn_password:                        '{{ lookup("file",     inventory_dir + "/personal_credentials/svn/password") }}'
